akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1096,8 +1090,9 @@ trait FlowOps[+Out, +Mat] {
   def withAttributes(attr: Attributes): Repr[Out, Mat]
 
   /** INTERNAL API */
-  private[scaladsl] def andThen[U](op: StageModule): Repr[U, Mat]
+  private[scaladsl] def andThen[T](op: SymbolicStage[Out, T]): Repr[T, Mat] =
+    via(SymbolicGraphStage(op))
```

Comment:
real beauty here, andThen is just a sugar now on top of via. Subclasses (Source, Flow) only need to provide a viaMat implementation now.